integration-tests/src/test/java/tachyon/shell/TfsShellTest.java

```diff
@@ -506,14 +506,17 @@ public void locationNotExistTest() throws IOException {
 
   @Test
   public void lsrTest() throws IOException, TachyonException {
-    // clear the loginUser
-    Whitebox.setInternalState(LoginUser.class, "sLoginUser", (String) null);
     MasterContext.getConf().set(Constants.SECURITY_GROUP_MAPPING,
         IdentityUserGroupsMapping.class.getName());
 
     FileInfo[] files = new FileInfo[4];
     String testUser = "test_user_lsr";
-    System.setProperty(Constants.SECURITY_LOGIN_USERNAME, testUser);
+    // clear the loginUser and re-login with new user
```

Comment:
Thank @dongche for the fix, a minor comment is we can move `user login` to a private method.